libs/build_gsl.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -eux
+
+name="gsl"
+version=${1:-${STACK_gsl_version}}
+
+compiler=$(echo $HPC_COMPILER | sed 's:/:-:g')
+mpi=$(echo $HPC_MPI | sed 's:/:-:g')
```

Comment:
MPI is not used.  Please remove this line.

Reply:
As with the change in stack_ufs_weather_ci.yaml and others, this change came from my merge of https://github.com/kgerheiser/hpc-stack/tree/feature/gsl into my branch.  @aerorahul please talk with @kgerheiser about any changes to this file.  I did not add this file or code and am not comfortable changing it.